Assets/Scripts/Pathfinding/Path_AStar.cs

```diff
@@ -121,7 +121,7 @@ public Path_AStar(World world, Tile tileStart, Tile tileEnd, string objectType =
             {
                 Path_Node<Tile> neighbor = edge_neighbor.node;
 
-                if (ClosedSet.Contains(neighbor) == true)
+                if (ClosedSet.Contains(neighbor))
```

Comment:
Quill18 wrote it like that to make it easier to read. But I agree ==true on a bool is itch inducing.